Makefile

```diff
@@ -1,6 +1,6 @@
 $(info Running PCP-STATSD makefile)
 
-TARGET_EXEC ?= a.out
+TARGET_EXEC ?= pcp-statsd.out
```

Comment:
Please drop the .out extension, just pcp-statsd is fine.

Also this binary will not load until user either drops `/usr/local/lib` into `/etc/ld.so.conf.d/` or puts that into env variable:

    LD_LIBRARY_PATH=/usr/local/lib ./build/pcp-statsd.out

Reply:
I wrote that in read.md FAQ section

Reply:
.out dropped from binary by 9913b60

Reply:
.out dropped from bianry in metric examples by 1f355f8